.github/workflows/python-build.yml

```diff
@@ -0,0 +1,78 @@
+name: Python Build + Pypi Upload
+
+on:
+  push:
+    branches: [master]
+    tags: ["v*.*.*"]
```

Comment:
It will build on every pull request, on every commit in master AND on every version tagged in master.
Build on pull request ensures that a pull request doesn't break python binding, before accepting merge request.
Build on commits ensure that a commit doesn't break python binding.
Build on tagged version do the release build and publish on pypi.

Reply:
Thanks. It seems I have a wrong understanding of of how exactly a tag can come to be.